frontend/src/features/properties/map/acquisition/add/AddAcquisitionForm.tsx

```diff
@@ -34,14 +35,35 @@ export const AddAcquisitionForm = React.forwardRef<
   const regionTypes = getOptionsByType(API.REGION_TYPES);
   const acquisitionTypes = getOptionsByType(API.ACQUISITION_TYPES);
   const acquisitionPhysFileTypes = getOptionsByType(API.ACQUISITION_PHYSICAL_FILE_STATUS_TYPES);
+  const [showDiffMinistryRegionModal, setShowDiffMinistryRegionModal] = React.useState<boolean>(
+    false,
+  );
+  const [ignoreRegionWarning, setIgnoreRegionWarning] = React.useState<boolean>(false);
+  const isMinistryRegionDiff = (values: AcquisitionForm): boolean => {
+    const selectedPropRegions = values.properties.map(x => x.region);
+    return (
+      (selectedPropRegions.length &&
+        (selectedPropRegions.indexOf(Number(values.region)) === -1 ||
+          !selectedPropRegions.every(e => e === selectedPropRegions[0]))) ||
+      false
+    );
+  };
```

Comment:
@devinleighsmith and @Maheshbabu28 - can we double check this is the intended logic as laid out in Confluence. I recall that BAs mentioned that the region check should be against the 1st property (not all). If that's the case then this logic could be simplified/reworked. Comments?

Reply:
![image](https://user-images.githubusercontent.com/102620827/192413089-91e4cecb-d0a2-4e29-b72c-b1fd2a98e44f.png)
As per confluence this is for multiple property(ies)